modules/deploy/pages/deployment-option/cloud/azure-private-link.adoc

```diff
@@ -0,0 +1,20 @@
+= Configure Azure Private Link with the Cloud API
+:description: Set up Azure Private Link to securely access Redpanda Cloud.
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+The Redpanda Azure Private Link service provides secure access to Redpanda Cloud from your own VPC. Traffic over Azure Private Link does not go through the public internet because a Private Link connection is treated as its own private Azure service. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.
```

Comment:
Is this page meant to be part of the PR? It doesn't seem related?

Reply:
Correct, not supposed to be part of the PR. I'll try to clean it uo